.github/mergify.yml

```diff
@@ -1,3 +1,6 @@
+# Note: We do not use the rebase strategy to merge PRs, because that
+# loses information needed by changelog-d to associate commits with PRs.
```

Comment:
It's pretty sad: I'm not a fan of merge commits. But I guess, we have no choice (short of dropping changelog-d or teaching it new tricks)...

Reply:
Mergify can use commit templates, so future work might be to define one for this case to ensure the PR is mentioned.

Reply:
Oh, that would be awesome!